template/package.json

```diff
@@ -11,13 +12,13 @@
         "test:ci": "vitest run --reporter=junit --outputFile.junit=./results/junit.xml",
         "test-dev": "vitest",
         "coverage": "vitest run --coverage",
-        "lint": "eslint src",
+        "lint": "eslint src --ext .js,.jsx,.ts,.tsx",
         "lint-fix": "npm run lint -- --fix",
         "audit": "npm audit --production --registry https://registry.npmjs.org --parseable",
         "format-code": "prettier --write \"src/**/*.{js,jsx,ts,tsx,json,css,scss,md}\"",
         "format-code-on-commit": "npm run format-code && git add",
         "pull-translations": "phrase pull --access_token $PHRASEAPP_ACCESS_TOKEN && npm run merge-translations",
-        "merge-translations": "nodenv ./mergeTranslations.js",
+        "merge-translations": "node mergeTranslations.js && rimraf src/features/translations/temp",
```

Comment:
Hmm, we could move the cleanup of the temp folder into the merge script itself to have everything together. We would not need an additional library too and keep the dependencies to a minimum.

Reply:
I updated the script so that it performs the deletion of the `temp` directory and dropped the `rimraf` dependency again.